Shared/Samples/Apply scheduled updates to preplanned map area/ApplyScheduledUpdatesToPreplannedMapAreaView.swift

```diff
@@ -0,0 +1,200 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyScheduledUpdatesToPreplannedMapAreaView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether the alert for
+    /// scheduled updates is presented.
+    @State private var updatesAlertIsPresented = false
+    
+    /// A Boolean value indicating whether the alert for
+    /// no available updates is presented.
+    @State private var noUpdatesAlertIsPresented = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        MapView(map: model.map)
+            .errorAlert(presentingError: $error)
+            .task {
+                do {
+                    try await model.setUp()
+                } catch {
+                    self.error = error
+                }
+            }
+            .onChange(of: model.updatesInfo == nil) { _ in
+                if let updatesInfo = model.updatesInfo {
+                    // Handle the updates info from the offline map sync task.
+                    handleUpdatesInfo(updatesInfo)
+                }
```

Comment:
```suggestion
                guard let updatesInfo = model.updatesInfo else { return }
                // Handle the updates info from the offline map sync task.
                handleUpdatesInfo(updatesInfo)
```


Not a big deal, but this may make more sense with a guard statement. Up to you!